ONSdigital/census-rm-actionexporter-service
# Remove DLQ queues, retry logic and add missing queue

# Motivation and Context
There was a race condition between the startup of the ActionExporter service and the Action Scheduler, which was causing problems with some queues being missing - either Action.Field or the DLQs (legacy) that the ActionExporter was expecting to find, causing warnings in the logs.

# What has changed
Removed the DLQs and retry logic. Added the Action.Field queue and exchange binding.

# How to test?
Build and run the ATs. To test thoroughly, re-terraform a brand new GCP environment and ensure that the warnings have gone and all the queues are set up OK... but it was an intermittent fault.

# Links
Trello: https://trello.com/c/VHEyfavC/847-defect-dlqs-and-retry-logic-should-be-removed-from-action-exporter
